src/EA.Weee.DataAccess.Tests.DataAccess/StoredProcedure/GetAatfEvidenceSummaryTotalsTests.cs

```diff
@@ -272,6 +272,72 @@ public async Task Execute_GivenAatfWithManyApprovedNotes_DataShouldBeReturned()
             }
         }
 
+        [Fact]
+        public async Task Execute_GivenCreateNoteWithStartDateOutsideComplianceYear_NotNullNoteShouldBeReturned()
```

Comment:
I think we can remove these tests as they don't test what we want for this class.